db/seeds.rb

```diff
@@ -26,8 +26,8 @@
 LoanQuestionSet.connection.schema_cache.clear!
 LoanQuestionSet.reset_column_information
 
-LoanQuestionSet.find_or_create_by(id: 2, division: Division.root,
```

Comment:
Why is division removed here? Automated change?

Reply:
I dropped and repopulated my db and got an error from the seeds file

After checking, there was no association between LoanQuestionSet and Division. Also, the seeds file is not always run which would make it harder to catch this

Reply:
Ok. Thanks.